charts/kube-prometheus-stack/templates/alertmanager/ingressperreplica.yaml

```diff
@@ -12,7 +13,7 @@ metadata:
 items:
 {{ range $i, $e := until $count }}
   - kind: Ingress
-    apiVersion: {{ include "kube-prometheus-stack.ingress.apiVersion" . }}
+    apiVersion: {{ include "kube-prometheus-stack.ingress.apiVersion" $root }}
```

Comment:
Instead of creating another variable, doesn't replacing `.` with `$` work?

Reply:
You are absolutely right, I removed unnecessary variable and it works as expected.